bpdm-orchestrator-api/src/main/kotlin/org/eclipse/tractusx/orchestrator/api/model/ResultState.kt

```diff
@@ -22,5 +22,17 @@ package org.eclipse.tractusx.orchestrator.api.model
 enum class ResultState {
     Pending,
     Success,
-    Error
+    Error,
+    NaturalPersonError,
+    BpnErrorMissingParent,
+    BpnErrorNotFound,
+    BpnErrorTooManyOptions,
+    MandatoryFieldValidationFailed,
+    BlacklistCountryPresent,
+    InvalidSpecialCharacters,
+    MandatoryFieldMissing,
+    BpnlChanged,
+    UnclearEntity,
+    UnknownSpecialCharacters,
+    MatchBasedOnProvidedNameOrIdentifier
```

Comment:
Result state indicate a high-level distinction between success, error and in-progress. This is not the place to put specific error types. For this, you can use the TaskErrorType in the same package